app/routes/users/components/UserProfile.js

```diff
@@ -363,6 +362,18 @@ export default class UserProfile extends Component<Props, UpcomingEventsProps> {
             ) : (
               <LoadingIndicator loading />
             )}
+            <h3>Dine tidligere arrangementer ({previousEvents.length})</h3>
+            {loading ? (
+              <LoadingIndicator margin={'20px auto'} loading />
+            ) : (
+              <ListEvents
+                events={previousEvents
+                  .filter(e => e.userReg.pool !== null)
+                  .filter(e => e.userReg.presence !== 'NOT_PRESENT')
```

Comment:
@edvardvb is this what you were thinking of?

Reply:
Yup!